addons/base_vat/models/res_partner.py

```diff
@@ -59,6 +59,17 @@
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
+    vat_validation_type = fields.Selection(selection_add=[('european_vat', 'European Vat')])
```

Comment:
Due to the fact that we have already merged the Mexican one, validation on this code, you are expecting in the l10n_mx module add the mx validation, right?